content/200-concepts/100-components/01-prisma-schema/06-relations/410-referential-actions/index.mdx

```diff
@@ -159,6 +159,9 @@ The following table shows which referential action each database supports.
 | MongoDB††  | **✔️**  | **✔️**   | **✔️**   | **✔️**  | **❌**     |
 
 - † On [InnoDB](https://dev.mysql.com/doc/refman/5.6/en/innodb-introduction.html#:~:text=InnoDB%20is%20a%20general%2Dpurpose,clause%20creates%20an%20InnoDB%20table.), `SetDefault` is ignored and is silently replaced with `NoAction` instead.
+  On MySQL 8.0+, when a `SetDefault` referential action is triggered, a foreign key violation error is raised.  
+  On older versions, like MySQL 5.6+, creating or altering a table with a `SetDefault` referential action is not possible at all.
+  We therefore advise against using `SetDefault` at all on MySQL.
```

Comment:
`Restrict` and `NoAction` are identical (aliases) in:
- `mysql`
- `cockroachdb`

In the other databases that support both actions, the're behaviorally equivalent only when executing a single statement in a transaction (but I would probably not include this last info, as it may be a source of confusion for the users).

Reply:
Do we know exactly which version of MySQL has the transition between failing and having the same behaviour as `NoAction` and `Restrict`?

I'd want to rewrite that part of the note as something like:

>  In MySQL versions X.X and later, the `SetDefault` referential action has the same behavior as `NoAction` or `Restrict`. In previous versions, creating or altering a table with a `SetDefault` referential action is not possible at all.

once we know the correct version.

Reply:
8.x as far as we know.